recipes/area-detector-handlers/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "area-detector-handlers" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "676cec048d2d41136d758fcdfe94a53aca551d2d3c267daf33c4a112a5662f18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dask
+    - entrypoints
+    - h5py
+    - pandas
+    - tifffile >=2020.8.25
+
+test:
+  imports:
+    - area_detector_handlers
+    - area_detector_handlers.handlers
+  requires:
+    - pytest
+  commands:  # [not win]
```

The selector here isn't doing anything.
```suggestion
  commands:
```